services/input-service.js

```diff
@@ -75,6 +75,27 @@ class InputService {
     }
 
     getBlockchain(options) {
+        const blockchainName = options.blockchain?.name;
```

Comment:
The same code appears in index.js, can you create this to be function in some service and then use that code in index.js and input-service.js?